### PR TITLE
Two small edit data fixes

### PR DESCRIPTION
**Pull request by mwiencek**
Commits seem self-explanatory.

### CODE REVIEW

**Comment by mwiencek** [comment]
I didn't investigate where the encoding issue came from. I just assumed it had something to do with text vs. jsonb.

File: lib/MusicBrainz/Server/Data/Edit.pm [hunk 1]
```diff
@@ -76,7 +77,7 @@ sub _new_from_row
         expires_time => $row->{expire_time},
         auto_edit => $row->{autoedit},
         status => $row->{status},
-        raw_data => $row->{data},
+        raw_data => decode('utf-8', $row->{data}),
```
Comment (gentlecat) [inline lib/MusicBrainz/Server/Data/Edit.pm]: This one is different from `$data` value from above, right?

    my $data = JSON::XS->new->utf8->decode($row->{data});

**Comment by gentlecat** [comment]
LGTM

File: lib/MusicBrainz/Server/Data/Edit.pm (hunk 1 above)
Comment (uklauer) [inline lib/MusicBrainz/Server/Data/Edit.pm]: More or less, the `$data` is a Perl hash representation of what is contained in the `$row->{data}` JSON string.

**Comment by uklauer** [comment]
I can only imagine that DBI does some special casing for `text` but not for `jsonb` columns, but not sure either.

**Pull request merged**